backend/driver/driver_test.go

```diff
@@ -44,7 +44,8 @@ func Test_Driver(t *testing.T) {
 		Context:      context.Background(),
 		JobNamespace: "fms-lm-eval-service-system",
 		JobName:      "evaljob-sample",
-		ConfigMap:    "configmap",
```

Comment:
Do we not need the `ConfigMap` entry here anymore?

Reply:
good catch, let me remove it